metadoc-core/src/main/protobuf/metadoc.proto

```diff
@@ -4,14 +4,23 @@ package metadoc.schema;
 
 message Position {
     string filename = 1;
-    int32 start = 2;
-    int32 end = 3;
+    reserved 2, 3;
+    Range range = 4;
```

Comment:
OK, makes sense. Yes, we can use extension to make the API nicer and limit the impact of future protobuf changes.